deploy/services/README.md

```diff
@@ -73,6 +84,17 @@ Please replace the same with your server's hostname.
   cp config/services.env.template config/services.env
   ```
 
+* Start PostgreSQL and run ThingsBoard install.
```

Comment:
The data and log directories of thingsboard need specific permissions

```bash
sudo chown -R 799:799 data/thingsboard
sudo chown -R 799:799 log/thingsboard
```
Please see [this page](https://github.com/thingsboard/thingsboard/issues/5683)